### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by Tshepo45**

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Tshepo45 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Linters passing
:heavy_check_mark: Correct Gitflow

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add a description to your pull request 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -74,70 +64,230 @@ <h1 class="heading-title">I'm Tshepo Makama<br> Glad to see <br> you!</h1>
...
+          <p> Languages </p>
+          <img src="image/drop-down.png" alt="arrow-down" />
+        </li>
+      </ul>
+
+      <ul class="programming-lang">
+        <li class="hero-links"> <img class="icon" src="/image/Frame9.png" alt="JavaScript icon"/> </li>
+        <li class="hero-links"> <img class="icon" src="/image/Frame10.png" alt="HTML5 icon"/> </li>
+        <li class="hero-links"> <img class="icon" src="/image/Frame11.png" alt="CSS3 icon"/> </li>
+      </ul>
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply index.html]: Fixed

File: index.html [hunk 2]
```diff
@@ -74,70 +64,230 @@ <h1 class="heading-title">I'm Tshepo Makama<br> Glad to see <br> you!</h1>
...
+        About
+        <br>Myself
+      </h2>
+      <p class="about-description">
+        Hello I’m a software developer! I can help <br> you build a product,
+        feature or website. <br> Look through some of my work and <br> experience! If
+        you like what you see and <br> have a project you need coded, don’t <br>
+        hestiate to contact me.
+      </p>             
+      <p class="connect"><a href="#">LET'S CONNECT</a></p>
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply index.html:221]: ## Optional
- [ ] I highly suggest that you work on this before merging

File: index.html [hunk 3]
```diff
@@ -74,70 +64,230 @@ <h1 class="heading-title">I'm Tshepo Makama<br> Glad to see <br> you!</h1>
...
-        <li>
-          <img src="/image/Ellipse4.png" alt="Ellipse4">
-         </li>
+        <li> <img src="/image/Ellipse4.png" alt="Ellipse4"> </li>
       </ul>
+    </div>
+  </section> 
 
+ <!--work section-->
+<section id="work-section">
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply index.html:103]: ## Optional
- [ ] Please fix this as well before merging

File: index.html [hunk 4]
```diff
@@ -74,70 +64,230 @@ <h1 class="heading-title">I'm Tshepo Makama<br> Glad to see <br> you!</h1>
...
+  <div class="container">
+    <figure class="snap-shoot-one">
+      <img class="project1-photo" src="/Portfolio-Mobile/image/Snap2.png" alt="project's image" />
+    </figure>
+    <div class="Project-header">
+      <h4>Tonic</h4>
+      
+      <ul class="project-description">
+        <li> <img class="project1-text" src="image/Text4.png" alt="CANOPY"/> </li>
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply index.html]: ## Required
- [ ] Please address this still image doesn't show

File: index.html [hunk 5]
```diff
@@ -74,70 +64,230 @@ <h1 class="heading-title">I'm Tshepo Makama<br> Glad to see <br> you!</h1>
...
-          <img src="/image/Ellipse4.png" alt="Ellipse4">
-         </li>
+        <li> <img src="/image/Ellipse4.png" alt="Ellipse4"> </li>
       </ul>
+    </div>
+  </section> 
 
+ <!--work section-->
+<section id="work-section">
+  <div class="container">
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply index.html:104]: Fixed

File: index.html [hunk 6]
```diff
@@ -74,70 +64,230 @@ <h1 class="heading-title">I'm Tshepo Makama<br> Glad to see <br> you!</h1>
...
+    <figure class="snap-shoot-one">
+      <img class="project1-photo" src="/Portfolio-Mobile/image/Snap2.png" alt="project's image" />
+    </figure>
+    <div class="Project-header">
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply index.html]: ## Optional
- [ ] Please fix this before merging

File: index.html [hunk 7]
```diff
@@ -74,70 +64,230 @@ <h1 class="heading-title">I'm Tshepo Makama<br> Glad to see <br> you!</h1>
...
+      <p class="about-description">
+        Hello I’m a software developer! I can help <br> you build a product,
+        feature or website. <br> Look through some of my work and <br> experience! If
+        you like what you see and <br> have a project you need coded, don’t <br>
+        hestiate to contact me.
+      </p>             
+      <p class="connect"><a href="#">LET'S CONNECT</a></p>
+      
+
+      <ul class="media-icons">
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply index.html:224]: Fixed

File: README.md [hunk 8]
```diff
@@ -1,16 +1,16 @@
-# Portfolio-Mobile
+# Portfolio Mobile 
 
 First mobile version portfolio project created with html templates and styling to built a responsive mobile version.
 
-Built With
+# Built With
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply README.md]: ## Optional
- [ ] I highly suggest that you follow this template given by the previous reviewer to update your README

**Review by DeliceLydia — APPROVED** [review]
## Hi @Tshepo45, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**